### DISCUSSION TITLE
Flow spec in dynamics mode

### DISCUSSION THREAD

**Discussion by svidille** [DanWBR/dwsim#588] [General]
Dear Dan and dear everybody,

When the flow spec is fixed in DWSIM dynamics mode, is it always mass flow? Is it possible to change to another (molar, liq volume, etc)?

Thanks,
Simone

**Comment by DanWBR** [comment]
@svidille if you change the flow rate in any of the three options, the other two will be updated accordingly.

**Comment by svidille** [comment]
Hi, Dan

Thanks for your answer. Your help is really appreciated. I see what you mean, I was trying to change the outlet stream, but it seems that is only valid for the inlet.

Best wishes,
Simone

**Comment by DanWBR** [comment]
@svidille you're welcome, but please do not create too many new discussions in a day, as I have to go through all of them just to check if they are bugs or just mistakes from your side. One discussion per day is ideal and enough, as I'm alone on this.
